openapi/src/main/scala/endpoints/openapi/DocumentedUrls.scala

```diff
@@ -0,0 +1,61 @@
+package endpoints
+package openapi
+
+/**
+  * Interpreter for [[algebra.DocumentedUrls]]
+  */
+trait DocumentedUrls extends algebra.DocumentedUrls {
+
+  type QueryString[A] = DocumentedQueryString
+
+  case class DocumentedQueryString(parameters: List[DocumentedParameter])
+
+  case class DocumentedParameter(name: String, required: Boolean) // TODO Schema
+
+  def combineQueryStrings[A, B](first: QueryString[A], second: QueryString[B])(implicit tupler: Tupler[A, B]): QueryString[tupler.Out] =
+    DocumentedQueryString(first.parameters ++ second.parameters)
+
+  def qs[A](name: String)(implicit value: QueryStringParam[A]): QueryString[A] =
+    DocumentedQueryString(List(DocumentedParameter(name, required = true)))
+
+  def optQs[A](name: String)(implicit value: QueryStringParam[A]): QueryString[Option[A]] =
+    DocumentedQueryString(List(DocumentedParameter(name, required = false)))
+
+  type QueryStringParam[A] = Unit
```

Comment:
Yes it does (cf [spec](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#parameter-object)), but I skipped that for a later iteration. (cf the TODO line 13)